controllers/users.go

```diff
@@ -29,6 +29,8 @@ func GetUsersHandler(c echo.Context) error {
 		users[i].Improve()
 	}
 
+	models.Log("Users listed", "Info", "mnewman")
```

Comment:
Levels should be defined as "constants" on the models so this line should look something like:
```
models.Log("Users listed", models.InfoLogLevel, "mnewman")
```
So in case of a developer typo the compiler will be complaining instead of finding the bug on prod :)